test/gtest/ucp/test_ucp_perf.cc

```diff
@@ -21,12 +21,14 @@ class test_ucp_perf : public ucp_test, public test_perf {
 public:
     enum {
         VARIANT_TEST_TYPE,
-        VARIANT_ATOMIC_MODE
+        VARIANT_ATOMIC_MODE,
+        VARIANT_WIREUP_MODE = VARIANT_ATOMIC_MODE,
     };
 
     enum {
         ATOMIC_CPU = 1,
-        ATOMIC_DEVICE
+        ATOMIC_DEVICE,
+        WIREUP_ALL_TO_ALL
```

Comment:
instead try to extend test of tag xfer to always enable this. some xfer tests run on all cuda mem combinations

Reply:
@yosefe I added testing to xfer but I keep it for now for more testing, can revert in the end on squash if you prefer.